libs/clients/reputation/client.go

```diff
@@ -332,3 +333,26 @@ func (c *HTTPClient) UpsertReputationSummary(ctx context.Context, paymentID, geo
 
 	return nil
 }
+
+type reputationSummaryPatchRequest struct {
+	VerifiedWallet bool `json:"verifiedWallet"`
+}
+
+// UpdateReputationSummary calls the reputation summary update endpoint with the values.
+func (c *HTTPClient) UpdateReputationSummary(ctx context.Context, paymentID string, verifiedWallet bool) error {
```

Comment:
looks very similar to upsert reputation summary.  is there a way we can combine these two into a wholistic "ReputationSummary" type that has geo and verified state, and whatever else we need in the future?

Reply:
The upset performs a create if not exist on the reputation side where as this endpoint will error if the reputation does not exist which is what we want in this case. The upsert can only be called once with the geo and will fail there after plus a few other things. Agreed we should look to simplify the rep summary going forward possibly using events.